bundle/android/strings.xml

```diff
@@ -348,9 +348,6 @@ translators: %s: Select control option value e.g: "Auto, 25%". -->
     <string name="gutenberg_native_slash_inserter_results" tools:ignore="UnusedResources">Slash inserter results</string>
     <string name="gutenberg_native_some_blocks_have_additional_settings_tap_the_settings_icon_on_the" tools:ignore="UnusedResources">Some blocks have additional settings. Tap the settings icon on the bottom right of the block to view more options.</string>
     <string name="gutenberg_native_start_writing" tools:ignore="UnusedResources">Start writing…</string>
-    <string name="gutenberg_native_take_a_photo" tools:ignore="UnusedResources">Take a Photo</string>
```

Comment:
This string seems to be only used in the native version, so these changes should be reverted. Not sure why the script in charge of updating the localizations has removed them, I'd need to investigate it further 😞 .

Reply:
Fixed in https://github.com/wordpress-mobile/gutenberg-mobile/pull/6204/commits/1b95e55629e4c9af7c0b17d14186222af0a596b9.